### PR TITLE
fix(devnet): generate a fixed set of public and private keys for devnet

### PR DESCRIPTION
**Pull request by antiyro**

### CODE REVIEW

File: crates/node/src/service/block_production.rs [hunk 1]
```diff
@@ -79,7 +79,10 @@ impl Service for BlockProductionService {
                     .context("Building genesis block from devnet config")?;
 
                 block_import
-                    .add_block(genesis_block, BlockValidationContext::new(backend.chain_config().chain_id.clone()))
+                    .add_block(
+                        genesis_block,
+                        BlockValidationContext::new(backend.chain_config().chain_id.clone()).trust_class_hashes(true),
```
Comment (cchudant) [inline crates/node/src/service/block_production.rs:84]: if you add that here, can you also add it in devnet/src/lib.rs in the tests?

**Review by cchudant — APPROVED** [review]
do we want:
- a mode where the keys are always random (?)
- a cli arg to change the seed?

i can't see any use cases for those though

File: crates/node/src/service/block_production.rs (hunk 1 above)
Comment (antiyro) [reply crates/node/src/service/block_production.rs:84]: yes I'll also remove any from_random usage on the tests to match what we have in the rest of the codebase

Comment (cchudant) [reply crates/node/src/service/block_production.rs:84]: i dont think there are any :)
we use the DevnetKeys struct directly to access the secrets

**Review by Trantorian1 — APPROVED** [review]

**Pull request merged**